lemur/plugins/lemur_acme/ultradns.py

```diff
@@ -96,12 +96,14 @@ def _has_dns_propagated(name, token, domain):
             for txt_record in rdata.strings:
                 txt_records.append(txt_record.decode("utf-8"))
     except dns.exception.DNSException:
-        metrics.send("has_dns_propagated_fail", "counter", 1)
+        function = sys._getframe().f_code.co_name
+        metrics.send("{}.fail".format(function), "counter", 1)
```

Comment:
to be consistent, it would be great if we could use f-strings for visibility:
`metrics.send(f"{function}.fail", "counter", 1)`

https://cito.github.io/blog/f-strings/